### PR TITLE
Make the repository a Julia package

### PR DESCRIPTION
**Pull request by Azzaare**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Panquesito7
```
Comment (Panquesito7) [inline LICENSE:3]:
```suggestion
Copyright (c) 2021 The Algorithms & contributors
```

File: docs/src/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+```@meta
+CurrentModule = TheAlgorithms
+```
+
+# TheAlgorithms
+
+Documentation for [TheAlgorithms](https://github.com/TheAlgorithms/TheAlgorithms.jl).
```
Comment (Panquesito7) [inline docs/src/index.md:7]:
```suggestion
Documentation for [TheAlgorithms](https://github.com/TheAlgorithms/Julia).
```

**Review by Panquesito7 — COMMENTED** [review]
I did a quick review, and I found 2 minor issues (for now).

**Comment by Azzaare** [comment]
This PR should also address #6

**Review by arubhardwaj — DISMISSED** [review]
I think it should be all good since @Azzaare is a pro!

**Review by arubhardwaj — APPROVED** [review]
Cool. Ignoring CI failure for macOS.

**Comment by Azzaare** [comment]
For MacOS on nightly build (also the problem comes from a highly popular dependency, DifferentialEquations.jl, so I am pretty sure it will be fixed eventually ^^)

**Comment by arubhardwaj** [comment]
@Azzaare, maybe a stupid question, but can we remove macOS thing from CI? Will it have any bad impact?

**Comment by Azzaare** [comment]
Removing macOS yes. Removing it only for nightly build, not really.

**Review by Panquesito7 — COMMENTED** [review]
Thanks so much, @Azzaare! 😄

**Pull request merged**